### PR TITLE
rewrite README

### PR DESCRIPTION
**Pull request by francois-vincent**

### CODE REVIEW

**Comment by francois-vincent** [comment]
CI fails because #152 is not merged !

File: README.md [hunk 1]
```diff
@@ -1,59 +1,123 @@
...
     fab --list
 
-Or see list of all available tasks [here](tasks.md).
+General fabric command format:
+
+    PYTHONPATH=<path_to_fabric_navitia>:<path_to_configuration> python -u -m fabric --keepalive=20 pseudo-task1:params pseudo-task2:params task1:params task2:params ... taskn:params
+
+or
+
+    PYTHONPATH=<path_to_fabric_navitia>:<path_to_configuration> fab --keepalive=20 pseudo-task1:params pseudo-task2:params task1:params task2:params ... taskn:params
```
Comment (pbougue) [inline README.md]: Can you precise the goal of keepalive?

Comment (pbougue) [inline README.md]: It would also be nice to explain the distinction between pseudo-tasks and tasks

Comment (pbougue) [inline README.md]: My bad for pseudo tasks (maybe just a `see below for pseudo tasks` is enough)

File: README.md [hunk 2]
```diff
@@ -1,59 +1,123 @@
...
+**restart_jormungandr**: restart jormungandr on all servers.
 
-    fab dev prepare_upgrade
-    fab dev upgrade_tyr
-    fab dev launch_rebinarization_upgrade # long time to wait !
-    fab dev upgrade_kraken
-    fab dev upgrade_jormungandr
-    
-Note : you can use the variable env.nb_thread_for_bina in the definition of the environment to parallelize binarizations.
+**update_all_configurations**: redeploy all configurations.
```
Comment (pbougue) [inline README.md]: A litlle more details would be good, on what conf is managed (by instance, jormun, tyr)

File: README.md [hunk 3]
```diff
@@ -1,59 +1,123 @@
...
 
-* Complete removal of an instance on all components:
 
-    ```fab dev remove_instance:fr-idf```
+Management
+----------
+Some important management tasks:
 
-* Do the upgrade on the dev environnment:
+**restart_kraken**: restart all krakens on all servers.
```
Comment (pbougue) [inline README.md]: the potential interests of those 2 restart might be good (conf, after a crash, ...)

File: fabfile/tasks.py [hunk 4]
```diff
@@ -201,6 +201,9 @@ def broadcast_email(kind, status=None):
 
 @task
 def update_tyr_step(time_dict=None, only_bina=True, up_confs=True):
+    # TODO only_bina is highly error prone
```
Comment (pbougue) [inline fabfile/tasks.py]: Why do you leave that TODO?
It's a good point, but do we want a warning to the user, or else?

File: README.md [hunk 5]
```diff
@@ -1,59 +1,123 @@
...
-Note : you can use the variable env.nb_thread_for_bina in the definition of the environment to parallelize binarizations.
+**update_all_configurations**: redeploy all configurations.
 
-prod only, use ws1 and eng1
+**update_instance**:  deploy a new coverage or deploy it again. (param=coverage name)
 
-    fab prod use_upgraded_version
-    
-prod only, upgrade other ws and eng when prod validated
+**launch_rebinarization**: launch a binarization on a single coverage, based on last dataset. (param=coverage name)
```
Comment (pbougue) [inline README.md]: How about the bina of a new dataset, like you asked recently?

File: README.md (hunk 1 above)
Comment (pbougue) [inline README.md]: Done

**Comment by pbougue** [comment]
Niiice PR, feeling really more comfortable after reading this if I have to use fabric :+1:

File: fabfile/tasks.py (hunk 4 above)
Comment (francois-vincent) [inline fabfile/tasks.py]: No we want to refactor this later

File: README.md (hunk 5 above)
Comment (francois-vincent) [inline README.md]: there's no task for that !

File: README.md (hunk 2 above)
Comment (francois-vincent) [inline README.md]: all confs *really* means all

File: README.md (hunk 3 above)
Comment (francois-vincent) [inline README.md]: Some details added

File: README.md (hunk 5 above)
Comment (pbougue) [inline README.md]: ok, I guess this is the role of tyr, not fabric :)

**Pull request merged**